Automattic/woocommerce-payments
# (Privacy) Fixed some newly added PII keys were not redacted

Fixes #6600 

#### Changes proposed in this Pull Request
Some PII info is not redacted in the API logs, so I've added them to the API key logs.









#### Testing instructions





- Checkout to this branch
- Place an order
- Goto `WooCommerce > Settings > Scheduled Actions`
- Run `wcpay_track_update_order` scheduled action
- Go to the latest log and check for the latest log
- Find the log for `https://public-api.wordpress.com/wpcom/v2/sites/%s/wcpay/tracking/order`
- Verify that the added keys values are now redacted.

-------------------


**Post merge**
